WooCommerce/Classes/ViewModels/CardPresentPayments/PaymentCaptureCelebration.swift

```diff
@@ -0,0 +1,31 @@
+import AudioToolbox
+import UIKit
+
+/// Plays a sound and provides taptic feedback when a payment capture has been completed successfully
+/// https://www.youtube.com/watch?v=ewRjZoRtu0Y
```

Comment:
Why this link? We aren't using this sound, are we?

Reply:
Nah. It was a joke. Maybe it was not a funny joke. I will remove it.